tmva/tmva/test/CMakeLists.txt

```diff
@@ -27,8 +27,12 @@ if(dataframe)
     # RReader
     ROOT_ADD_GTEST(rreader rreader.cxx LIBRARIES ROOTVecOps TMVA ROOTDataFrame)
     # Tree inference system and user interface
-    ROOT_ADD_GTEST(branchlessForest branchlessForest.cxx LIBRARIES TMVA)
-    ROOT_ADD_GTEST(rbdt rbdt.cxx LIBRARIES ROOTVecOps TMVAUtils)
+    # Commented out right now because RBDT doesn't provide low-level interfaces
+    # since the sync with FastForest. Only the construction from XGBoost models
+    # is supported. If this feature becomes necessary to implement, one can
+    # resurrect these tests.
+    # ROOT_ADD_GTEST(branchlessForest branchlessForest.cxx LIBRARIES TMVA)
+    # ROOT_ADD_GTEST(rbdt rbdt.cxx LIBRARIES ROOTVecOps TMVAUtils)
 endif()
```

Comment:
You need to add the `rbdt_xgboost.py` file to the CMakeLists for it to be included. In the CMake command you should also state the dependency on `xgboost` so that ctest runs the corresponding fixture beforehand to check the package is present in the environment